README.md

```diff
@@ -360,6 +362,92 @@ When using flakes, switch to new configurations as you do for the
 whole system (e. g. `nixos-rebuild switch --flake <path>`) instead of
 using the `home-manager` command line tool.
 
+### For Unprivileged Setup
+
+If you would like to set up home-manager as an unprivileged user:
+1.  Place `flake.nix` under `~/.config/nixpkgs/`
+
+    ```Nix
+    {
+      description = "Home manager configuration of jane"
+    
+      inputs = {
+        home-manager.url = "github:nix-community/home-manager";
+        # Optional, get in sync with the desired nixpkgs source
+        nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+        home-manager.inputs.nixpkgs.follows = "nixpkgs";
+      };
+    
+      outputs = { home-manager, ... }:
+        let
+          system = "x86_64-linux";
+          username = "jdoe";
+        in {
+          homeManagerConfiguration.${username} = home-manager.lib.homeManagerConfiguration {
+            configuration = import ./home.nix;
+            inherit system username;
+            homeDirectory = "/home/${username}";
+            # # Pass extra arguments into home.nix using
+            # extraSpecialArgs = {
+            #   key = value;
+            # };
+          };
+      };
+    }
+    ```
+    *   If you would like to use the `release-20.09` branch, change the home-manager input url to `github:nix-community/home-manager/release-20.09` and nixpkgs url to `github:NixOS/nixpkgs/nixos-20.09`.
+
+2.  On `~/.config/nixpkgs`, build the configuration by
+
+    ```console
+    $ nix build .#homeManagerConfiguration.jdoe.activationPackage
+    ```
+
+    The build result will show up in the `./result` directory
+
+3.  If you satisfied with the result, install and switch to the generation by
+
+    ```console
+    $ ./result/activate
+    ```
+
+If you are using the master branch of home-manager, switch to a new generation based on the flake by
```

Comment:
```suggestion
2.  Since the release `21.05`, building the configuration that resides in `~/.config/nixpkgs` is as simple as

    ```console
    $ home-manager switch --flake '~/.config/nixpkgs#jdoe'
    ```

    where `jdoe` is a configuration specified in the flake file `~/.config/nixpkgs/flake.nix`.
```